templates/model.txt

```diff
@@ -1,6 +1,10 @@
+// This model was generated by Lumber. However, you remain in control of your models.
+// Learn how to enrich this model here: https://docs.forestadmin.com/documentation/v/v4/reference-guide/models/enrich-your-models
```

Comment:
I think we should make all our comments 1-liners, so as to remain very light. The idea behind this is: 
1. It takes less room vertically, so less visually cumbersome
2. If a dev needs more info on a specific part of the model, he reads the comment and scroll right to use the link if necessary.

So here it would be: 
_This model was generated by Lumber. However, you remain in control of your models. Learn how here: https://docs.forestadmin.com/documentation/v/v4/reference-guide/models/enrich-your-models_

Reply:
(github formats it on 2 lines, but the idea is to have all of it in 1 line)

Reply:
Well when I read a comments I prefer it to be visible fully without need to scroll. So here your comment will be too long and so I think it's great on 2 lines

Reply:
Final version (on 2 lines):
This model was generated by Lumber. However, you remain in control of your models.
Learn how here: https://docs.forestadmin.com/documentation/v/v4/reference-guide/models/enrich-your-models